apps/new/widget/page/project/Metadata.jsx

```diff
@@ -94,13 +94,36 @@ const EditButton = ({ item }) => {
   );
 };
 
+const HyperLink = styled.a`
```

Comment:
Do we have an existing component like this already?

Reminds me of button outline

Reply:
You're right actually, for a second there I thought we couldn't edit the colors on the "Button" 😅